lib/index.js

```diff
@@ -1000,6 +1002,8 @@ _.use = use(Stream, _);
 addToplevelMethod('of', function (x) {
     return _([x]);
 });
+Stream.prototype[fl.of] = Stream.prototype.of;
+_[fl.of] = _.of;
```

Comment:
I don't think we need this. Fantasy land defines `of` as a method of the *type representative* of the type, which must be the constructor. `_` is not the constructor.

I think we'll need to expose the constructor to be fully compliant. That'll require a bit more work than needs to be done in this PR. Let's get this merged first.

Reply:
I think I see what you mean.

In the expression `v.ap(A.of(x => x))`, though, wouldn't `A.of` typically be represented as `_.of` in typical highland usage?

That would lead me to believe it belongs on both `Stream` (for correctness) _and_ `_` (for idiomatic functionality).

Reply:
That's fair.